packages/cozy-client/src/models/document/locales/fr.json

```diff
@@ -47,12 +49,16 @@
       "other_revenue": "Autres revenues",
       "gradebook": "Bulletin de notes",
       "student_card": "Carte d'étudiant",
+      "school_attendance_certificate": "Certificat de scolarité",
       "resume": "CV",
       "motivation_letter": "Lettre de motivation",
       "other_work_document": "Autre document de travail",
       "health_book": "Carnet de santé",
       "health_certificate": "Certificat de santé/vaccination",
+      "work_disability_recognition": "Reconnaissance de handicap - RQTH",
+      "pregnancy_medical_certificate": "Certificat de grossesse",
       "national_health_insurance_card": "Carte vitale",
+      "national_health_insurance_right_certificate": "Attestation de droits de l'Assurance Maladie (vitale)",
```

Comment:
Why the "(vitale)"? It sounds like a mix between "cette assurance est vitale" and "carte vitale"

Reply:
Because the website of " l'assurance maladie" adds is also themselves, because it is used used in common language : "attestation de carte vitale". But I think adding "(carte vitale)" would be missleading with the "carte vitale" itself.